src/fidesops/schemas/privacy_request.py

```diff
@@ -118,6 +118,13 @@ class ReviewPrivacyRequestIds(BaseSchema):
     request_ids: List[str] = Field(..., max_items=50)
 
 
+class DenyPrivacyRequests(ReviewPrivacyRequestIds):
+    """Pass in a list of privacy request ids and rejection reason"""
+
+    reason: Optional[str]
+    request_ids: List[str] = Field(..., max_items=50)
```

Comment:
since this extends `ReviewPrivacyRequestIds`, don't think `request_ids` property is needed here.

Reply:
good catch. removed